### DISCUSSION TITLE
[Question] Is there a way to get core file?

### DISCUSSION THREAD

**Discussion by Marco-Sulla** [pypa/cibuildwheel#2271] [Q&A]
### Description

I have a segfault during pytests, and I can't reproduce it in local. Is there a way to put on artifacts core files?

### Build log

https://github.com/Marco-Sulla/python-frozendict/actions/runs/9487232241/job/26143336956?pr=103

**Comment by Czaki** [comment]
You men that your local compilation do not fail. Or you downloaded the artifacts (after disabling the tests) and they are working on your machine?

General possibility could be to enable core files in `docker` using `CIBW_BEFORE_ALL` and modify test commend to put core file in `wheeldir` in container (command need to end with success)
